packages/abap-deploy-config-inquirer/src/translations/abap-deploy-config-inquirer.i18n.json

```diff
@@ -55,101 +55,101 @@
                     "message": "Package",
                     "messageTypeFilter": " (Type to filter matching records)",
                     "hint": "Select a package for the deployed application.",
-                    "sourceMessage": "More than {{numResults}} results, keep typing to narrow down"
+                    "sourceMessage": "More than {{numResults}} results. Keep typing to narrow down the list."
                 }
             },
             "transport": {
                 "common": {
                     "transportRequest": "Transport Request",
                     "transportRequestMandatory": "Transport Request (mandatory)",
-                    "provideTransportRequest": "Provide a transport request for your application"
+                    "provideTransportRequest": "Provide a transport request for your application."
                 },
                 "transportInputChoice": {
-                    "message": "How do you want to enter the transport request?"
+                    "message": "Enter Transport Request"
```

Comment:
Same as above, this prompt shows different options for how someone wants to enter the transport. So the premise of the questions needs to remain the same